src/accelerate/accelerator.py

```diff
@@ -299,10 +299,10 @@ def __init__(
             self.project_configuration.set_directories(project_dir)
         if mixed_precision is not None:
             mixed_precision = str(mixed_precision)
-            if mixed_precision not in PrecisionType:
-                raise ValueError(
-                    f"Unknown mixed_precision mode: {mixed_precision}. Choose between {PrecisionType.list()}"
-                )
+        else:
+            mixed_precision = os.environ.get("ACCELERATE_MIXED_PRECISION", "no")
```

Comment:
Should be `None` instead of `no`
```suggestion
        else:
            mixed_precision = os.environ.get("ACCELERATE_MIXED_PRECISION", None)
```